spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/Resilience4JCircuitBreaker.java

```diff
@@ -102,25 +102,49 @@ public Resilience4JCircuitBreaker(String id, String groupName,
 		this.bulkheadProvider = bulkheadProvider;
 	}
 
+	public Resilience4JCircuitBreaker(String id, String groupName,
+			io.github.resilience4j.circuitbreaker.CircuitBreakerConfig circuitBreakerConfig,
+			TimeLimiterConfig timeLimiterConfig, CircuitBreakerRegistry circuitBreakerRegistry,
+			TimeLimiterRegistry timeLimiterRegistry,
+			Optional<Customizer<io.github.resilience4j.circuitbreaker.CircuitBreaker>> circuitBreakerCustomizer,
+			Resilience4jBulkheadProvider bulkheadProvider) {
```

Comment:
You can just call `this(...)` passing `null` for the executor service

Reply:
Implemented as suggested.